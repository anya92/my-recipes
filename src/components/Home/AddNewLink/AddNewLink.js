import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router-dom';

const StyledButton = styled.div`
  text-align: center;
  margin: 20px 0;
  padding: 10px;
  a {
    font-weight: 700;
    font-size: 1.35rem;
    text-decoration: none;
    color: #333;
    // background-color: dodgerblue;
    // padding: 5px 20px;
    // border-radius: 5px;
  }
`;

const AddNewLink = () => (
  <StyledButton>
    <Link to="add">
      Add New
    </Link>
  </StyledButton>
);

export default AddNewLink;
