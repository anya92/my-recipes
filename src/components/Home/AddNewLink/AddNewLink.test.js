import React from 'react';
import { shallow } from 'enzyme';

import AddNewLink from './AddNewLink';

test('should render AddNewLink correctly', () => {
  const wrapper = shallow(<AddNewLink />);
  expect(wrapper).toMatchSnapshot();
});
