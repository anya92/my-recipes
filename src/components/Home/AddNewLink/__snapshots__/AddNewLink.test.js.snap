// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddNewLink correctly 1`] = `
<styled.div>
  <Link
    replace={false}
    to="add"
  >
    Add New
  </Link>
</styled.div>
`;
